docs/source/conf.py

```diff
@@ -0,0 +1,62 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'dlpack'
+copyright = '2022, DLPack contributors'
+author = 'DLPack contributors'
+
+# The full version, including alpha/beta/rc tags
+release = '0.6'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['breathe']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Breathe configuration ---------------------------------------------------
+
+# Tell breathe what to call the project. This is used to link XML files
+# generated by Doxygen by passing a command line option:
+# -Dbreathe_projects.<breathe_default_project>=</path/to/xml/index/directory>
+breathe_default_project = "dlpack"
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'pydata_sphinx_theme' # 'alabaster'
+
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ['_static']
```

Comment:
If there are no static files in this documentation build, please remove the `_static` path.
```suggestion
html_static_path = []
```

See here: https://github.com/readthedocs/readthedocs.org/issues/1776

Reply:
I just commented them out as @driazati suggested. So that if the docs ever evolve to use static files and/or templates, we can just uncomment these lines.